marimo/_ast/codegen.py

```diff
@@ -30,7 +30,17 @@ def _multiline_tuple(elems: Sequence[str]) -> str:
 
 
 def _to_decorator(config: Optional[CellConfig]) -> str:
-    if config is None or config == CellConfig():
+    if config is None:
+        return "@app.cell"
+
+    # Removed defaults. If the cell's config is the default config,
+    # don't include it in the decorator.
+    if config.disabled is False:
```

Comment:
Maybe there's a lint rule for this

```suggestion
    if not config.disabled:
```